docs/README.md

```diff
@@ -435,7 +435,7 @@ For detailed information see the documentation on [lit-html.polymer-project-org]
 To render a template from your element just define a `template()` method and return a `tagged template literal`.
 
 ```javascript
-export class MyElement extends TemplateElement {
```

Comment:
I think we should keep using "TemplateElement" for documentation. What do you think @quarkus ?

Reply:
@eddyloewen I think that was a search/replace mistake